texas-hearing-institute/listening_settings_new/components/ToggleGridButtonsComponent/ToggleButton.tsx

```diff
@@ -0,0 +1,18 @@
+import { Text, TouchableOpacity } from 'react-native';
+import styles from './ToggleButtonStyle';
+
+interface Props {
+    title: string;
+    isToggled: boolean;
+    onToggle: (newValue: boolean) => void;
+}
+
+const ToggleButton: React.FC<Props> = ({ title, isToggled, onToggle }) => {
```

Comment:
it's better styling to leave out the `React.FC` typing for functional components!